src/app/(proper_react)/redesign/(public)/LandingView.tsx

```diff
@@ -42,37 +43,79 @@ export const View = (props: Props) => {
             isHero
             eligibleForPremium={props.eligibleForPremium}
             signUpCallbackUrl={`${process.env.SERVER_URL}/redesign/user/dashboard/`}
+            eventId={{
+              cta: "clicked_get_scan_header",
+              field: "entered_email_address_header",
+            }}
           />
         </div>
         <div className={styles.heroImage}>
           <HeroImage {...props} />
         </div>
       </header>
 
-      <div className={styles.valuePropositionWrapper}>
+      <section className={styles.quoteWrapper}>
+        <div className={styles.quote}>
+          <h2>
+            {props.eligibleForPremium
+              ? props.l10n.getFragment("landing-premium-quote", {
+                  elems: {
+                    data_brokers: <span className={styles.emphasis} />,
```

Comment:
We have an element for emphasis :) (Likewise on line 68)

```suggestion
                    data_brokers: <em className={styles.emphasis} />,
```